nautobot/docs/development/apps/migration/ui-component-framework.md

```diff
@@ -0,0 +1,244 @@
+# Migrating to UI Component Framework
+A guide for app developers transitioning to the UI Component Framework.
+
+## Introduction
+
+This guide helps you migrate your existing Nautobot app views to use the new UI Framework. The framework provides a declarative approach to building object detail views, reducing boilerplate code while ensuring consistency across the platform.
+
+For complete UI Framework documentation, see: [Nautobot UI Framework Documentation](../../core/ui-component-framework.md)
+
+## Why Migrate?
+
+### Benefits
+- Reduced template maintenance
+- Consistent UI patterns across apps
+- Built-in responsive layouts
```

Comment:
I'd remove this one personally - there's nothing specifically more responsive about the new framework versus manual templates.